### PR TITLE
Add `operator.call()` and `operator.__call__()` for Python 3.11

### PR DESCRIPTION
**Pull request by lancelote**
`call()` and `__call__()` ware added to `operator` (`call` is also defined in `_operator`) module in Python 3.11

- [What’s New In Python 3.11 - Improved Modules - operator](https://docs.python.org/3.11/whatsnew/3.11.html#operator)
- [bpo-44019 - `operator.call/operator.__call__`](https://bugs.python.org/issue44019)

I hope I've used `ParamSpec` correctly here. Would greatly appreciate any corrections 🙇

### CODE REVIEW

**Comment by lancelote** [comment]
Wrong `ParamSpec` import, I'm fixing it 🏃

File: stdlib/operator.pyi [hunk 1]
```diff
@@ -177,3 +180,7 @@ def itruediv(__a: Any, __b: Any) -> Any: ...
 def __itruediv__(a: Any, b: Any) -> Any: ...
 def ixor(__a: Any, __b: Any) -> Any: ...
 def __ixor__(a: Any, b: Any) -> Any: ...
+
+if sys.version_info >= (3, 11):
+    def call(__obj: Callable[_P, _T], *args: _P.args, **kwargs: _P.kwargs) -> _T: ...
+    def __call__(obj: Callable[_P, _T], *args: _P.args, **kwargs: _P.kwargs) -> _T: ...
```
Comment (AlexWaygood) [inline stdlib/operator.pyi:186]: Nit: maybe use `_R` for the return-type `TypeVar`? (Otherwise, LGTM!)

File: stdlib/_operator.pyi [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # In reality the import is the other way around, but this way we can keep the operator stub in 2and3
```
Comment (Akuli) [inline stdlib/_operator.pyi:1]: This comment is no longer up to date, as Python 2 and 3 stubs have been separated. We should probably change this to match runtime, but in a separate PR.

File: stdlib/operator.pyi [hunk 3]
```diff
@@ -177,3 +181,7 @@ def itruediv(__a: Any, __b: Any) -> Any: ...
 def __itruediv__(a: Any, b: Any) -> Any: ...
 def ixor(__a: Any, __b: Any) -> Any: ...
 def __ixor__(a: Any, b: Any) -> Any: ...
+
+if sys.version_info >= (3, 11):
+    def call(__obj: Callable[_P, _R], *args: _P.args, **kwargs: _P.kwargs) -> _R: ...
+    def __call__(obj: Callable[_P, _R], *args: _P.args, **kwargs: _P.kwargs) -> _R: ...
```
Comment (Akuli) [inline stdlib/operator.pyi:187]: mypy doesn't support `ParamSpec` yet:

```suggestion
    def call(__obj: Callable[_P, _R], *args: _P.args, **kwargs: _P.kwargs) -> _R: ...  # type: ignore
    def __call__(obj: Callable[_P, _R], *args: _P.args, **kwargs: _P.kwargs) -> _R: ...  # type: ignore
```

Comment (AlexWaygood) [reply stdlib/operator.pyi:187]: Any idea why it doesn't seem to be causing any errors in the CI tests? I was surprised it seemed to work as well as it did.

Comment (Akuli) [reply stdlib/operator.pyi]: The CI doesn't run Python 3.11 yet, so everything under `if sys.version_info >= (3, 11):` is untested for now.

**Review by Akuli — APPROVED** [review]

File: stdlib/operator.pyi (hunk 3 above)
Comment (AlexWaygood) [reply stdlib/operator.pyi]: Hah, makes sense.

**Pull request merged**